watchdog/readconfig.go

```diff
@@ -49,21 +52,10 @@ func (ReadConfig) Read(hasEnv HasEnv) WatchdogConfig {
 
 	cfg.faasProcess = hasEnv.Getenv("fprocess")
 
-	readTimeout := parseIntValue(hasEnv.Getenv("read_timeout"))
-	writeTimeout := parseIntValue(hasEnv.Getenv("write_timeout"))
-
-	cfg.execTimeout = time.Duration(parseIntValue(hasEnv.Getenv("exec_timeout"))) * time.Second
-
-	if readTimeout == 0 {
-		readTimeout = 5
-	}
-
-	if writeTimeout == 0 {
-		writeTimeout = 5
-	}
+	cfg.readTimeout = parseIntOrDurationValue(hasEnv.Getenv("read_timeout"), time.Second*5)
```

Comment:
I would set the default 5 second (for both params) in a const rather than having a magic value inline.

Reply:
Agree. I will update that.

Reply:
@alexellis @johnmccabe What should be the default value for `exec_timeout`? Right now it is `0` second.

Reply:
default is 5?
https://github.com/openfaas/faas/blob/ed6a09f71fa465465a5f91c7acf965d6ac565a5d/watchdog/readconfig.go#L57-L63